ingenialink/ethercat/servo.py

```diff
@@ -4,6 +4,7 @@
 
 import ingenialogger
 
+
```

Comment:
Remove blank line

Reply:
don't we have a black or ruff formatter for this project?

Reply:
Yes, we have Black. Format check is ok with or without this blank line.